### PR TITLE
updates british

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

updates British to be beh'er.
Added bare words to the list, before mans list was kinda lacking.

## Why It's Good For The Game

Everyone loves bri'ish.

I used primary and secondary sources for gathering the sounds, its more roadman than chav but that's not a problem innit.

## Changelog
:cl:
refactor: Refactors england.
/:cl:

### CODE REVIEW

**Comment by TeomanTheGreat** [comment]
Rather based innit?

File: strings/british_talk.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        "murder": ["shank", "bang"],
+        "murdered": ["shanked", "banged"],
+        "stab": ["shank", "bang"],
+        "stabbed": ["shanked", "banged"],
+		"robust": ["shank", "bang"],
+        "robusted": ["shanked", "banged"],
+        "thief": ["robber", "burglar"],
+		"centcom": "royal family",
+		"man": ["block", "lad", "fam", "fella"],
+		"friend": ["mate", "g", "fam"],
```
Comment (Cenrus) [inline strings/british_talk.json]:
```suggestion
		"friend": ["mate", "g", "fam", "blad"],
```

Comment (PowerfulBacon) [reply strings/british_talk.json]: blud*

**Comment by PowerfulBacon** [comment]
I dont know which words are british and which words are normal so its kinda hard, but I think I got a good amount

File: strings/british_talk.json [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        "captain": "queen",
+        "cap": "queen",
+        "kill": ["shank", "bang"],
+        "killed": ["shanked", "banged"],
+        "murder": ["shank", "bang"],
+        "murdered": ["shanked", "banged"],
+        "stab": ["shank", "bang"],
+        "stabbed": ["shanked", "banged"],
+		"robust": ["shank", "bang"],
+        "robusted": ["shanked", "banged"],
```
Comment (Cenrus) [inline strings/british_talk.json:42]:
```suggestion
        "robusted": ["shanked", "banged", "clobbered"],
```

Comment (Pontenerd) [reply strings/british_talk.json:42]: I've lived in England most of my life and I've not heard bang/banged before except when referring to someone being shot

File: strings/british_talk.json [hunk 3]
```diff
@@ -0,0 +1,118 @@
+{
+
+    "words": {
+        "id": "BBM Pin",
```
Comment (ghost) [inline strings/british_talk.json:4]: This is stupid.

**Comment by Effemu** [comment]
br*tish 🤮

**Comment by eeSPee** [comment]
Yet chips is not crisps, and fries is not chips.

Wha' world d'ya live 'n, 'overnoh!

File: strings/british_talk.json (hunk 3 above)
Comment (PowerfulBacon) [reply strings/british_talk.json:4]: wtf how could you be so offensive

Comment (Pontenerd) [reply strings/british_talk.json:4]: Jesus Christ bacon that reference is ANCIENT

**Comment by utilbird** [comment]
Decrease gun damage for increased knife damage

File: strings/british_talk.json (hunk 3 above)
Comment (ghost) [reply strings/british_talk.json:4]: back in my day.... based

**Comment by PowerfulBacon** [comment]
> 
> 
> Decrease gun damage for increased knife damage

holy shit that is genius

**Comment by DatBoiTim** [comment]
Ah yes finally.

A Bacon Meme PR

**Comment by sircope** [comment]
now make british, swedish and elvis a trait 😳

File: strings/british_talk.json [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+		"conflict": "beef",
+		"beautiful": "peng",
+		"cute": "peng",
+		"cool": "sick",
+		"good": "sick",
+		"amazing": "sick",
+		"bayonet": "shank",
+		"thanks": "cheers",
+		"thank you": "cheers",
+		"i am": "man is"
```
Comment (PowerfulBacon) [inline strings/british_talk.json]:
```suggestion
		"i am": "man is",
		"fries": "chips"
```

File: strings/british_talk.json (hunk 3 above)
Comment (Kmc2000) [inline strings/british_talk.json:4]:
```suggestion
        "id": "BBM Pin",
        "ass": "batty",
        "crazy": "mad",
        "cool": "calm",
        "awesome": "lit",
        "many":"bare",
        "fries": "chips",
        "sidewalk": "pavement",
        "australia": "penal colony",
```

Comment (PowerfulBacon) [reply strings/british_talk.json:4]: Thing is already in the list. Also shouldn't it be pavement?

File: strings/british_talk.json [hunk 5]
```diff
@@ -0,0 +1,127 @@
+{
+
+    "words": {
+        "id": "BBM Pin",
+        "ass": "batty",
+        "crazy": "mad",
+        "cool": "calm",
```
Comment (Kmc2000) [inline strings/british_talk.json:7]:
```suggestion
        "cool": "calm",
        "big": "thick",
        "huge":"wham",
        "weed":"gas",
        "car":"whip",
        "friends":"mandem",
        "soccer":"footy",
        "drink":"pint",
        "cigarette":"cig",
        "call":"bell",
        "mom":"mum",
        "lyrics":"bars",
        "bar":"pub",
```

**Review by Kmc2000 — COMMENTED** [review]
Make it so the terminal full stop of a sentence is just "still."

File: strings/british_talk.json (hunk 5 above)
Comment (PowerfulBacon) [reply strings/british_talk.json:7]: Knife is already in as shank

File: strings/british_talk.json [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+        "t": "'",
+		"g ": "' ",
+		"s ": "z "
+    },
+	"appends": [
+		" innit bruv",
+		" br-r-r-rap!",
+		" man",
+		" blud",
+		" innit"
```
Comment (PowerfulBacon) [inline strings/british_talk.json]:
```suggestion
		" innit",
		" still"
```

File: strings/british_talk.json [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+		"places": "ends",
+		"person": "badman",
+		"traitor": "roadman",
+		"changeling": "roadman",
+		"nukie": "roadman",
+		"want to": "wanna",
+		"wants to": "wanna",
+		"gun": "gat",
+		"pistol": "gat",
+		"knife": "shank",
```
Comment (PowerfulBacon) [inline strings/british_talk.json]:
```suggestion
		"knife": ["shank", "skeng"],
```

File: strings/british_talk.json [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+		"beautiful": "peng",
+		"cute": "peng",
+		"cool": "sick",
+		"good": "sick",
+		"amazing": "sick",
+		"bayonet": "shank",
+		"thanks": "cheers",
+		"thank you": "cheers",
+		"i am": "man is",
+		"fries": "chips"
```
Comment (TheFakeElon) [inline strings/british_talk.json:123]:
```suggestion
		"fries": "chips",
		"euro": "pound"
```

**Comment by Vexylius** [comment]
PB you are an absolute gem of this community, god bless you

**Comment by ToasterBan** [comment]
best changelog I have ever seen

File: strings/british_talk.json (hunk 3 above)
Comment (eeSPee) [reply strings/british_talk.json:4]: Should be 'loicense'

**Comment by ike709** [comment]
I feel like the json files should be cached the first time someone speaks british.

**Comment by PowerfulBacon** [comment]
<s>ill make the felinid virus a bit less crappy too</s>

**Comment by PowerfulBacon** [comment]
@ike709 Its already cached by strings() nevermind

**Review by utilbird — APPROVED** [review]

**Pull request merged**